src/Console/Command/PostExporter.php

```diff
@@ -88,20 +88,19 @@ public function handle()
 			'exporter' => true
 		];
 
-		if ($job_id) {
-			// Load the export job
-			$job = $this->exportJobRepository->get($job_id);
 
-			$this->session->setUser($job->user_id);
-			// Merge the export job filters with the base filters
-			if ($job->filters) {
-				$filters = array_merge($filters, $job->filters);
-			}
+		// Load the export job
```

Comment:
Job_id is a mandatory parameter in the cli now. It'll fail if you don't provide it

Reply:
This was a bug hidden by the fact that we always send a job id . But it was bad anyway, and was hiding a bug in CliController where id was null